duckduckgo/iOS
# Remove autofill beta badge

Task/Issue URL: https://app.asana.com/0/414235014887631/1200632147004925/f
Tech Design URL:
CC:

**Description**:

This PR removes the `Beta` badge found in the settings screen.

**Steps to test this PR**:
1. Open settings and check that there is no reference to `Beta`
1. Use the double length pseudo-language setting on the app scheme and verify that the cell clips its text as you would expect, as it has a title and detail label on it when you're signed in



**Orientation Testing**:


**Device Testing**:


**OS Testing**:


---
###### Internal references:
[Software Engineering Expectations](https://app.asana.com/0/59792373528535/199064865822552)
[Technical Design Template](https://app.asana.com/0/59792373528535/184709971311943)
**When ready for review, remember to post the PR in MM**
